internal/testing/s3_test.go

```diff
@@ -166,11 +166,13 @@ func TestS3List(t *testing.T) {
 	}
 
 	objects := []string{
-		"sample.jpg",
-		"photos/2006/January/sample.jpg",
-		"photos2/2006/February/sample3.jpg",
-		"photos/2006/February/sample2.jpg",
-		"photos2/2006/February/sample4.jpg",
+		"/foo/bar",
```

Comment:
I don't think the files we upload through s3 should start with `/`. It's not illegal but I think it's also not how you commonly organise files on S3. 

e.g.
https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-prefixes.html
https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-folders.html